budgetportal/webflow/urls.py

```diff
@@ -12,4 +12,9 @@
         views.provincial_infrastructure_project_detail,
         name="provincial-infra-project-detail",
     ),
+    url(
+        r"^infrastructure-projects/provincial/search/$",
```

Comment:
Let's put this at `/api/v1/infrastructure-projects/provincial`
search is just an operation it supports. And we want to separate it from the HTML views.

Reply:
url changed

https://github.com/vulekamali/datamanager/pull/190/commits/506e3913fdaee7ec173ecd6042251c18ebf9cba4